acceptance/kubernetes/kind/kubernetes.go

```diff
@@ -400,10 +400,17 @@ func (k *kindCluster) TaskInfo(ctx context.Context) (*types.TaskInfo, error) {
 			return nil, err
 		}
 
+		envVars, err := k.envVars(ctx, t.namespace, tr.Status.PodName, s.Container)
+		if err != nil {
+			return nil, err
+		}
+
+		// TODO: Grab env vars here (maybe from the pod?)
```

Comment:
Is this todo stale now?
```suggestion
```

Reply:
Yes. Removed it. Thanks for catching this.